### PR TITLE
Patch 3

### PR DESCRIPTION
**Pull request by espyra88**
In order to help users, i provide a quick and easy to use template, to create alert-rules!

What is this feature?

This is a doc contribution to help user to easily create alert-rules through api. I had a lot of issues before finally complete this working template. Caring is sharing!

Why do we need this feature?

Absolutely no ready-to-use/helping template on the net, nor the documentation.

Who is this feature for?

Every users using API call to create alert-rules

Which issue(s) does this PR fix?:

Fixes #

Special notes for your reviewer:

Please check that:

[ X] It works as expected from a user's perspective.
 If this is a pre-GA feature, it is behind a feature toggle.
[W ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

I add some issues trying to edit from patch-2, so i closed the previous one (https://github.com/grafana/grafana/pull/80815), put the change in the right place then pulling a new one.

### CODE REVIEW

File: docs/sources/alerting/set-up/provision-alerting-resources/_index.md [hunk 1]
```diff
@@ -40,6 +40,91 @@ Choose from the options below to import your Grafana Alerting resources.
 
    For more information on the Alerting Provisioning HTTP API, refer to [Alerting provisioning HTTP API][alerting_provisioning].
 
+   Example ready-to-use:
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/provision-alerting-resources/_index.md:43]:
```suggestion
   Here is a ready-to-use template for alert rules:
```

File: docs/sources/alerting/set-up/provision-alerting-resources/_index.md [hunk 2]
```diff
@@ -40,6 +40,91 @@ Choose from the options below to import your Grafana Alerting resources.
...
+   
+   #### Alert-rules template
```
Comment (brendamuir) [inline docs/sources/alerting/set-up/provision-alerting-resources/_index.md:45]:
```suggestion
   #### Alert rules template
```

**Comment by brendamuir** [comment]
One more thing - could you run prettier on your branch please? yarn install && yarn run prettier:write - this is to get rid of some lint issues blocking the ci - thanks!

**Comment by brendamuir** [comment]
ok, i ran prettier. can you accept my changes and we can get this merged? :)

**Comment by espyra88** [comment]
that's done, thanks a lot!

**Review by brendamuir — APPROVED** [review]

**Pull request merged**